app/src/main/java/io/multy/model/entities/wallet/Wallet.java

```diff
@@ -173,15 +169,15 @@ public double getBtcDoubleValue() {
     }
 
     public double getEthDoubleValue() {
-        return balance.equals("0") ? 0 : (getBalanceNumeric().longValue() / (EthWallet.DIVISOR).doubleValue());
+        return balance == null || balance.equals("0") ? 0 : (getBalanceNumeric().longValue() / (EthWallet.DIVISOR).doubleValue());
```

Comment:
unacceptable. if our balance == null so something wrong with saving to db and we should have exception in that case